cloud-img/README.md

```diff
@@ -0,0 +1,18 @@
+# Custom Image for Stackdriver Sandbox Project
+
+When the Dockerfile changes, here are the steps to follow in order to push it to GCR:
+
+Build the Dockerfile, tag the image, and push the image to the Container Registry.
```

Comment:
instead of naming the steps and then all the commands, you can name a step and show the command for that step, then do the same for the next step and so on.

Reply:
Ah, okay. It makes sense that it would be easier to follow that way!